.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "third_party/RapidWright"]
 	path = third_party/RapidWright
 	url = https://github.com/Xilinx/RapidWright.git
+[submodule "third_party/xilinx-unisims"]
+	path = third_party/xilinx-unisims
+	url = https://github.com/Xilinx/XilinxUnisimLibrary.git
```

Comment:
Maybe it is safer to reference SymbiFlow's fork? (we'll be sure nobody removes or pushforces it)

Reply:
Done